src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -36,6 +37,7 @@ const CheckBoxCustom = (props: {value: boolean; label: string}) => {
           marginRight: 5
         }}>
         <CheckBox
+          disabled={props.disabled || false}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line correctly uses the `disabled` prop in the `CheckBox` component. If `props.disabled` is not provided, it defaults to `false`, which means the checkbox will be enabled by default.



<!-- This is an auto-generated comment by OSS CodeRabbit -->